pkg/cmd/package/support/pack.go

```diff
@@ -0,0 +1,270 @@
+package support
+
+import (
+	"archive/zip"
+	"errors"
+	"fmt"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/question"
+	"github.com/OctopusDeploy/cli/pkg/util"
+	"github.com/OctopusDeploy/cli/pkg/util/flag"
+	"github.com/bmatcuk/doublestar/v4"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+const (
+	FlagId        = "id"
+	FlagVersion   = "version"
+	FlagBasePath  = "base-path"
+	FlagOutFolder = "out-folder"
+	FlagInclude   = "include"
```

Comment:
Upon seeing an 'include' parameter, I went looking for an 'exclude' parameter, but there isn't one.

Can we add one please? It's super useful to be able to do things like `zip myarchive.zip * -x .git`, so I'd guess that would apply here too?